cmd/cagent/options.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+var opts struct {
+	// Slice of bool will append 'true' each time the option
+	// is encountered (can be set multiple times, like -vvv)
+	OutputFile         string `short:"o" long:"output" description:"file to write the results"`
+	ConfigPath         string `short:"c" long:"config" description:"config file path"`
+	Verbose            string `short:"v" long:"verbose" default:"error" description:"log level – overrides the level in config file (values \"error\",\"info\",\"debug\")"`
```

Comment:
This is inconsistent. The field is called `Verbose` but everywhere else we refer to it as `LogLevel` and also the description calls it `log level`